### PR TITLE
Refactor regions, part 1

### PR DESCRIPTION
**Pull request by jasonbaldridge**
Added a Region class with name and polygon attributes so that we can extend it to include wikidata queries (and thus ensure that every region we use is supported throughout and localize all specifications for each region).

Updated all flags that rely on regions so that they use the defined regions and support message.

Slight tweak to extracting wikigeo: it adds '_osm' to the output path name if OSM data is included.

Finally, I ran run_all.sh and removed/fixed stuff that was out of sync. (We should probably change some of these so that they don't output files that are checked in.)

### CODE REVIEW

File: run_all.sh [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
-bazel build cabby/rvs:*
-bazel query cabby/rvs:* | xargs bazel test
-bazel-bin/cabby/rvs/generate_rvs --rvs_data_path "./cabby/geo/pathData/pittsburgh_geo_paths.gpkg" --save_instruction_path "./cabby/rvs/data/pittsburgh_instructions.json" 
+# Do we still need geo_computation?
+# bazel-bin/cabby/geo/geo_computation --orig_lat 40.749102 --orig_lon -73.984076 -dest_lat 40.748432 --dest_lon -73.982473
 
-# Wikidata
-bazel-bin/cabby/data/wikidata/extract_geofenced_wikidata_items --region Pittsburgh
+bazel-bin/cabby/geo/map_processing/map_processor --region $REGION_NAME --min_s2_level 18 --directory "/tmp/poiTestData/"
+bazel-bin/cabby/geo/sample_poi --region $REGION_NAME --min_s2_level 18 --directory "/tmp/poiTestData/" --path "/tmp/pathData/pittburgh_small_geo_paths.gpkg" --n_samples 1
```
Comment (tzufgoogle) [inline run_all.sh:12]: Region for sample_poi should b DC becuse Pittsburgh_small won' produce a path as it is too small.

**Review by tzufgoogle — CHANGES_REQUESTED** [review]
One small thing to change in th run file

File: run_all.sh (hunk 1 above)
Comment (jasonbaldridge) [reply run_all.sh]: Fixed!

**Review by tzufgoogle — APPROVED** [review]
LGTM

**Pull request merged**